template/{{cookiecutter.module_name}}/1.0/{{cookiecutter.module_name}}.smk

```diff
@@ -66,7 +86,9 @@ rule _{{cookiecutter.module_name}}_step_1:
     threads:
         CFG["threads"]["step_1"]
     resources:
-        mem_mb = CFG["mem_mb"]["step_1"]
+        **CFG["resources"]["step_1"]
```

Comment:
Are these changes reflected in the associated config file?

Reply:
Yes.